jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AccountResource.java

```diff
@@ -671,11 +672,11 @@ public Response getInvoicesForAccount(@PathParam("accountId") final UUID account
 
         final List<Invoice> invoices;
         if (unpaidInvoicesOnly) {
-            invoices = new ArrayList<Invoice>(invoiceApi.getUnpaidInvoicesByAccountId(accountId, startDate, endDate, tenantContext));
+            invoices = new ArrayList<Invoice>(invoiceApi.getUnpaidInvoicesByAccountId(accountId, startDate, endDate, tenantContext)); //TODO_1272:  withInvoiceItems not passed here as it is not possible to get unpaid invoices without retrieving invoice items. Maybe add a check at the start to disallow passing both the unpaidInvoicesOnly and withInvoiceItems flags?  
```

Comment:
Ideally, this api also allows to not fetch invoice items, but this is just an optimization issue, and hopefully the number of unpaid invoices is small, so fetching invoice items when not required is probably ok.

Reply:
Changed to code to add a check that disallows both `unpaidInvoicesOnly` and `includeInvoiceComponents` to be specified at once.